src/proto/h2/client.rs

```diff
@@ -205,14 +205,14 @@ where
     req_rx: ClientRx<B>,
 }
 
-impl<B> ClientTask<B>
-where
-    B: HttpBody + 'static,
-{
-    pub(crate) fn is_extended_connect_protocol_enabled(&self) -> bool {
-        self.h2_tx.is_extended_connect_protocol_enabled()
-    }
-}
+// impl<B> ClientTask<B>
```

Comment:
Is this no longer needed? I think it's how HTTP/2 does it's websocket upgrades, right?

Reply:
`conn::http2::ClientTask` is not public, and the only existing public call site is here.

https://github.com/hyperium/hyper/blob/9e8fc8fca195f470a82be5bfb2fd8019c044b97a/src/client/conn/mod.rs#L370

Reply:
Oh I see, so probably something to add back to the `http2::Connection` type.